dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -59,7 +59,7 @@ includeClientGSSCredentialInSubject=Add the client delegation credentials to sub
 includeClientGSSCredentialInSubject.desc=Specifies whether the client delegation credentials should be stored in a client subject.
 
 includeCustomCacheKeyInSubject=Add the custom cache key to subject
-includeCustomCacheKeyInSubject.desc=Specifies whether the custom cache key should be stored in a client subject and LTPA cookie.
+includeCustomCacheKeyInSubject.desc=Specifies whether the custom cache key should be stored in a client subject and LTPA cookie. If this property is set to true and the cache key for a user is not found in the authentication cache, the user will be required to log in again. Set this property to false when you use the SPNEGO feature to allow the security subject to be constructed from the LTPA cookie and the user registry. If the security subject is reconstructed from the user registry, there will be no Kerberos and GSS Credentials in the subject. If your LTPA cookie is used by more than one server, consider setting this property to false.
```

Comment:
suggested edit for ID guidelines:

Specifies whether the custom cache key is stored in a client subject and LTPA cookie. If this property is set to true and the cache key for a user is not found in the authentication cache, the user is required to log in again. Set this property to false when you use the SPNEGO feature to allow the security subject to be constructed from the LTPA cookie and the user registry. If the security subject is reconstructed from the user registry, no Kerberos or GSS credentials are included in the subject. If more than one server uses your LTPA cookie , consider setting this property to false.

Reply:
Done

Reply:
For some reason, I dont see any changes

Reply:
forgot to push it. Done now